MainModule/Client/UI/Default/Settings.lua

```diff
@@ -202,4 +217,4 @@ return function(data)
 		gTable = window.gTable
 		window:Ready()
 	end
-end
\ No newline at end of file
+end
```

Comment:
Why did you remove the new line at the end of the file?
It's a standard programming convention.
```suggestion
end

```

Reply:
I think github does this automaticly

Reply:
I don't remember doing that. It's either an accident or GitHub behavior.